### ISSUE TITLE
Update Accessibility guidelines reference

### ISSUE THREAD

**Issue by Ainali** [publiccodenet/standard#985]
Last week, WCAG was released in version 2.2. We should perhaps update the link on readability. However, just changing the 21 to 22 in the existing URL does not work, as it seems that v.2.2 is not in the quick reference yet. Should we change to https://www.w3.org/TR/WCAG22/#reading-level for now and then change to the quick reference later (as that one has some helpful links), or should we just wait?

**Comment by ericherman** [comment]
I suggest we wait for a bit, it's not urgent.

**Comment by Ainali** [comment]
It seems like it has been added to the quick reference now.
